docs/vSphere-prerequisites.md

```diff
@@ -0,0 +1,30 @@
+# Pre-configuration to create a Windows VM golden image:
```

Comment:
Rename the file to `vSphere-vm-golden-image.md`

Reply:
Ignore the above request and rename the file to `vsphere-prerequisites.md` as we don't seem to be proper casing other file names.